src/openms/source/ANALYSIS/ID/SiriusAdapterAlgorithm.cpp

```diff
@@ -540,36 +540,33 @@ namespace OpenMS
 
     void SiriusAdapterAlgorithm::logInSiriusAccount(String& executable, const String& email, const String& password) const
     {
```

Comment:
There should also be a check and e.g. throw of an exception because this is a public member.

Reply:
Is that required? The email and password need to be passed in the function call anyways?

Reply:
But it can be empty. This is not checked now

Reply:
if this doesn’t crash/error I guess it is fine

Reply:
If used without the check e.g. from pyopenms when empty strings are provided

Reply:
In that case it will print the output from sirius login that user name and/or password was incorrect.